### DISCUSSION TITLE
Archiving data

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#568] [Q&A] [answered]
@shaselton  @shaselton-usds  
Hi- qq. 

Any thoughts on plan sponsors that are trying to archive the MRF?  The final rule doesn't require archival of the data- pretty sure because it is updated monthly.

**Comment by RLTx1391** [comment]
I think I've heard anything from a few months to 2 years of retention mentioned. Archival isn't required as far as I can tell.

**Comment by shaselton-usds** [comment, marked as answer]
Retention requirement can be found on the technical clarification section on the [TiC website](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification). (question 37)
